otx/cli/builder/builder.py

```diff
@@ -208,10 +208,11 @@ def build_task_config(
         template_config.dump(str(workspace_path / "template.yaml"))
 
         # Create Data.yaml
-        data_subset_format = {"ann-files": None, "data-roots": None}
-        data_config = {"data": {subset: data_subset_format.copy() for subset in ("train", "val", "test")}}
-        data_config["data"]["unlabeled"] = {"file-list": None, "data-roots": None}
-        mmcv.dump(data_config, str(workspace_path / "data.yaml"))
+        if (workspace_path / "data.yaml").exists() is False:
```

Comment:
This is what I said in the comment above. Deleting this if statement and its contents also looks good.

Reply:
Agree with this and the above comments, I also prefer this option.